packages/fxa-settings/src/components/Settings/PageSecurityEvents/SecurityEvent.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+export function SecurityEvent({
+  name,
+  createdAt,
+  verified,
+}: {
+  name: string;
+  createdAt: number;
+  verified?: boolean;
+}) {
+  const createdAtDateText = Intl.DateTimeFormat('en-US', {
```

Comment:
Should this `en-US` be localized?

Reply:
@pdehaan would you mind filing a ticket for us to investigate? 🙏

Reply:
Converted to an issue as https://github.com/mozilla/fxa/issues/14803
Oddly, I don't see the Jira sync update on that issue yet, so if it doesn't appear by the time we merge this PR I'll manually file again in Jira.